test/test_modules.py

```diff
@@ -483,6 +483,22 @@ def test_pip_package(host):
     assert int(outdated['latest'].split('.')[0]) > 2
 
 
+def test_x509(host):
+    host.check_output('apt-get -y update')
+    host.check_output('apt-get -y install curl')
+    host.check_output(
+        'curl -so /tmp/crt https://letsencrypt.org/certs/isrgrootx1.pem.txt'
+    )
```

Comment:
I'd suggest using local (in docker container) certificates from /etc/ssl instead.